Makefile

```diff
@@ -32,6 +32,9 @@ check: test
 	# tests should fail if go fmt results in uncommitted code
 	git status --porcelain
 	/bin/bash -c 'test $$(git status --porcelain | wc -l) == 0'
+	go get github.com/gordonklaus/ineffassign
+	"${GOPATH}/bin/ineffassign" .
```

Comment:
That's being done in #179 but may take a while to land. Let's go with this for now, until #179 lands.